### DISCUSSION TITLE
Welcome to DARC Discussions!

### DISCUSSION THREAD

**Discussion by lidangzzz** [Project-DARC/DARC#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by markxu002** [comment]
good project！

**Reply by gm3609** [reply]
nice

**Reply by AlbertJee0** [reply]
nice project

**Reply by babbaa** [reply]
OoOo2247409f756366d3d231770c4e0108f5b659c6d08733dd75f1a86fc57c1b513a8

**Reply by SOTAND** [reply]
wow

**Reply by ab3lxiao** [reply]
great future ahead

**Reply by an189** [reply]
0x27C49e8DEa7bc8eB24106422F1366cdD2E1fdfB4

**Comment by LONGCHRNGLIANG** [comment]
🫰

**Comment by LittleKwai** [comment]
零
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Project-DARC/DARC"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年12月17日(星期天) 晚上9:17
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)





 
🫰

**Comment by LotsoY** [comment]
收到发自我的iPhone------------------ Original ------------------From: LONGCHRNGLIANG  ***@***.***>Date: Sun,Dec 17,2023 9:17 PMTo: Project-DARC/DARC ***@***.***>Cc: Subscribed ***@***.***>Subject: Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)
🫰

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/Project-DARC/DARC/discussions/16#discussioncomment-7876796",
"url": "https://github.com/Project-DARC/DARC/discussions/16#discussioncomment-7876796",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Reply by survivzhang** [reply]
11111111111

**Comment by z777za** [comment]
bump up

**Comment by Fuyaoqin** [comment]
当哥

**Reply by fanzhongrong** [reply]
nice

**Comment by pikacat2018** [comment]
党哥妈妈粉报到

**Comment by biticc** [comment]
good

**Comment by yundiel** [comment]
hello

**Comment by JAYzhang123** [comment]
gsdgs56d4gsd5gsなベルガモットオジビタからロピカなフアやャクヤなどがミドっぱ

**Comment by Empty0Qc** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by xiaoguge** [comment]
you are welcome.










执迷不悟
***@***.***


---Original---
From: ***@***.***&gt;
Date: Mon, Dec 18, 2023 01:07 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)




 
hello

**Comment by xiaoguge** [comment]
helloNice to meet you












执迷不悟
***@***.***


---Original---
From: ***@***.***&gt;
Date: Mon, Dec 18, 2023 01:07 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)




 
hello

**Comment by battleP** [comment]
LFG!

**Comment by xiaoguge** [comment]
gogogo












执迷不悟
***@***.***


---Original---
From: ***@***.***&gt;
Date: Mon, Dec 18, 2023 01:49 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)




 
LFG!

**Comment by Bravestarjo** [comment]
党哥你好……

**Comment by cosmic-latte** [comment]
mark

**Comment by Faitlux** [comment]
Mark too



---Original---
From: "Wang Yang ***@***.***&gt;
Date: Mon, Dec 18, 2023 02:09 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)




 
mark

**Comment by JonthanH** [comment]
Hello!

yundiel ***@***.***> 于 2023年12月17日周日 12:07写道：

**Comment by newplayman** [comment]
hi发自我的 iPhone在 2023年12月18日，02:46，JonthanH ***@***.***> 写道：﻿
Hello!



yundiel ***@***.***> 于 2023年12月17日周日 12:07写道：

**Comment by Levvercetti** [comment]
looking good!

**Comment by IsotankGGG** [comment]
Get it

**Comment by Churchillhuang** [comment]
Hi everyone

**Comment by fuchyu** [comment]
very good

**Reply by VentionWong** [reply]
Great!

**Comment by lastshot99** [comment]
Good project

**Comment by Codaqi** [comment]
Good luck🤞

**Comment by Tgoooooo** [comment]
泰国AKM

**Comment by buqunzzz** [comment]
哈哈哈哈在 2023年12月18日，10:00，Companysandwich ***@***.***> 写道：﻿
LFG~~

**Comment by 530475988** [comment]
hi  中国人驾到

**Comment by meistingray** [comment]
niubi

On Mon, Dec 18, 2023 at 10:23 AM The god of wealth ***@***.***>
wrote:

> hi 中国人驾到
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Project-DARC/discussions/16#discussioncomment-7880880>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6PAH75CMITHUIANPZY563YJ6SJZAVCNFSM6AAAAABAYC3ASGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQOBQHA4DA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Best Regards!

Shiwei Yin
M.R.,
Beijing Cornwall Vanda Tech. Co.,Ltd, Daiichi-Sankyo Company, Limited.
TEL:+81-080-8709-3574  +86-189-0026-0093

**Comment by shitou001** [comment]
你们他妈的别发了，手机响哥不停



发自我的iPhone


------------------ Original ------------------
From: The god of wealth ***@***.***&gt;
Date: Mon,Dec 18,2023 10:23 AM
To: Project-DARC/DARC ***@***.***&gt;
Cc: Subscribed ***@***.***&gt;
Subject: Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)





 
hi  中国人驾到

**Comment by BMaayhew** [comment]
cool

fuchyu ***@***.***>于2023年12月18日 周一09:26写道：

**Comment by lyd168** [comment]
项目运行了下，还要继续完善啊，加油！

**Comment by GYneverlose** [comment]
让这个世界安静一会行不行

**Comment by wsf3266** [comment]
非常好的项目，加油

**Comment by zihuixilai** [comment]
good

**Comment by Kkkinghe** [comment]
good

**Reply by Kkkinghe** [reply]
good

**Comment by Lh104621** [comment]
pretty

**Comment by gm3609** [comment]
gm

**Comment by Copgig** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by hook9518** [comment]
干就完了

**Comment by zblinken** [comment]
good

**Comment by laitpike** [comment]
加油

**Comment by axinii** [comment]
Noted with many thanks.

**Comment by 625598034d** [comment]
取消订阅



---Original---
From: ***@***.***&gt;
Date: Mon, Dec 18, 2023 11:18 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)




 Noted with many thanks. 
 
 On Mon, Dec 18, 2023, 11:14 laitpike ***@***.***&gt; wrote: 
 
 &gt; 加油 
 &gt; 
 &gt; — 
 &gt; Reply to this email directly, view it on GitHub 
 &gt; <https://github.com/orgs/Project-DARC/discussions/16#discussioncomment-7881210&gt;, 
 &gt; or unsubscribe 
 &gt; <https://github.com/notifications/unsubscribe-auth/A6HJ4RCJVN7442GBIQHECQDYJ6YIFAVCNFSM6AAAAABAYC3ASGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQOBRGIYTA&gt; 
 &gt; . 
 &gt; You are receiving this because you are subscribed to this thread.Message 
 &gt; ID: ***@***.***&gt; 
 &gt;

**Comment by cryptobongg** [comment]
牛逼

**Reply by Gxjx2017** [reply]
没搞懂呀

**Comment by ak28hs** [comment]
这到底是啥啊🥲
cryptobongg ***@***.***>于2023年12月18日 周一上午11:20写道：

**Comment by Vtalike** [comment]
Good

**Comment by alex-ffang** [comment]
LFG

**Comment by brocatbtc** [comment]
冲 比AVAV屌

**Comment by topmininglabs** [comment]
build together

**Comment by FreeAllDAO** [comment]
爱你哦

**Comment by ak28hs** [comment]
kenni7774 ***@***.***>于2023年12月18日 周一下午3:50写道：

**Comment by liupeilink** [comment]
怎么领取兄弟



发自我的iPhone


------------------ Original ------------------
From: kenni7774 ***@***.***&gt;
Date: Mon,Dec 18,2023 3:50 PM
To: Project-DARC/DARC ***@***.***&gt;
Cc: Subscribed ***@***.***&gt;
Subject: Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)





 
DARC

**Comment by buzilan** [comment]
错过了 sol 的bonk，可以看看 avax 的 coq，冲啊

---- Replied Message ----
| From | ***@***.***> |
| Date | 12/18/2023 15:54 |
| To | Project-DARC/DARC ***@***.***> |
| Cc | Subscribed ***@***.***> |
| Subject | Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16) |

怎么领取兄弟



发自我的iPhone


------------------ Original ------------------
From: kenni7774 ***@***.***&gt;
Date: Mon,Dec 18,2023 3:50 PM
To: Project-DARC/DARC ***@***.***&gt;
Cc: Subscribed ***@***.***&gt;
Subject: Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)






DARC

**Comment by ming06162** [comment]
怎么领取，没搞明白！

**Comment by AKAKAY04** [comment]
出去传销了 拉了300人

**Comment by lilixikun** [comment]
yes




冷相随
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Project-DARC/DARC"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年12月18日(星期一) 下午5:25
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)





 
11111111111

**Comment by INNO-COOL** [comment]
我已收到邮件，会尽快处理，有事电话联系。

**Comment by Decembb** [comment]
我是来占便宜的，我爱说实话发自我的 iPhone在 2023年12月18日，17:30，INNO-COOL ***@***.***> 写道：﻿
我已收到邮件，会尽快处理，有事电话联系。

**Reply by yundiel** [reply]
区块链扫地僧，好奇新事物。

**Comment by Lissonwong** [comment]
😂😂😅

Decembb ***@***.***>于2023年12月18日 周一21:34写道：

**Comment by zouqilaiY** [comment]
加油呀！

**Reply by xunai84** [reply]
加油

**Comment by hcy973747509** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by MonkeyShen** [comment]
我已收到

**Comment by Ethanwoohaha** [comment]
不懂

**Comment by Yoobos** [comment]
具体情况呢

**Comment by 2017ray** [comment]
Good Project. Hope everything goes well.

**Comment by wangliusen** [comment]
所有人都能收到？？？？



发自我的iPhone


------------------ Original ------------------
From: 2017ray ***@***.***&gt;
Date: Tue,Dec 19,2023 10:41 AM
To: Project-DARC/DARC ***@***.***&gt;
Cc: Subscribed ***@***.***&gt;
Subject: Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)





 
Good Project. Hope everything goes well.

**Comment by hldan124** [comment]
这个挺牛逼



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年12月19日 10:37 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***> |
| 主题 | Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16) |

具体情况呢

**Comment by jxyclxj** [comment]
good!

**Comment by wdyhms** [comment]
cool!

jxyclxj ***@***.***> 于2023年12月19日周二 11:03写道：

**Comment by Tgoooooo** [comment]
I’m here发自我的 iPhone在 2023年12月18日，09:25，BMaayhew ***@***.***> 写道：﻿
cool



fuchyu ***@***.***>于2023年12月18日 周一09:26写道：

**Comment by sffhmmmmnnn** [comment]
nice project

**Comment by Ashington258** [comment]
🙏🏿

**Comment by q2511058156** [comment]
请问立党老师，这个项目怎么部署呢，所需要的环境有哪些以及环境的版本号是多少呢？

**Comment by echobigdata** [comment]
党哥在做一个伟大的项目

**Comment by apple305** [comment]
好，收到



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年12月20日 16:09 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***> |
| 主题 | Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16) |

党哥在做一个伟大的项目

**Comment by vvck001** [comment]
只要能发财的项目，都是伟大的


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Project-DARC/DARC"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年12月20日(星期三) 下午4:12
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16)





 好，收到
 
 
 
 ---- 回复的原邮件 ----
 | 发件人 | ***@***.***&gt; |
 | 日期 | 2023年12月20日 16:09 |
 | 收件人 | ***@***.***&gt; |
 | 抄送至 | ***@***.***&gt; |
 | 主题 | Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16) |
 
 党哥在做一个伟大的项目

**Comment by saulmon1024** [comment]
收到

**Comment by m24927605** [comment]
收到

**Comment by zouxianwei1984** [comment]
strong togther



| |
邹县委
|




---- Replied Message ----
| From | ***@***.***> |
| Date | 12/21/2023 08:19 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | Re: [Project-DARC/DARC] Welcome to DARC Discussions! (Discussion #16) |

strong togther

**Comment by cryptoim** [comment]
nice project and when will it come out bro

**Comment by ggwp37** [comment]
您的邮件我已经收到

**Comment by Gxjx2017** [comment]
我也想知道怎么领取

**Comment by hey365** [comment]
what the hell it is ?

Gxjx2017 ***@***.***> 于2023年12月21日周四 20:11写道：

**Comment by Setcoinfree** [comment]
What is this email for?--Sent from Yandex.Mail for mobile21.12.2023, 13:53, "hey365" ***@***.***>:
what the hell it is ?



Gxjx2017 ***@***.***> 于2023年12月21日周四 20:11写道：

**Comment by lsy0911** [comment]
这是来自QQ邮箱的假期自动回复邮件。链接：https://pan.baidu.com/s/11TAra3iqThp3aT5Zwz61iQ 提取码：1949 复制这段内容后打开百度网盘手机App，操作更方便哦

**Comment by buddhakidd** [comment]
Hi guys, awaiting more details about this project!

**Reply by lidangzzz** [reply]
Sure, the whitepaper will be ready in 2024 Q1

**Reply by echobigdata** [reply]
太棒了，党哥

**Comment by tingdax** [comment]
收到哈

发自我的iPhone

在 2024年1月1日，上午1:39，babbaa ***@***.***> 写道：

﻿

OoOo2247409f756366d3d231770c4e0108f5b659c6d08733dd75f1a86fc57c1b513a8

**Comment by AgentLex** [comment]
perfect

**Comment by SkyCapital** [comment]
okay

Sab, 27 Jan 2024 pukul 23.06 AgentLex ***@***.***> menulis:

**Comment by rifxyzz** [comment]
wen airdrop xaaxa

Sab, 27 Jan 2024 pukul 23.04 AgentLex ***@***.***> menulis:

**Comment by JAYzhang123** [comment]
gsdgs56d4gsd5gsなベルガモットオジビタからロピカなフアやャクヤなどがミドっぱ

**Comment by hcy973747509** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by INNO-COOL** [comment]
我已收到邮件，会尽快处理，有事电话联系。

**Comment by HanbgCN** [comment]
With coding knowledge, don't know what could do for this project. Stay tune.

**Comment by Impeiqi** [comment]
good

**Comment by Halo1236** [comment]
👍

**Comment by tonyteacher4** [comment]
0x0C4EC1eb342252448EA2cED4B0f2e58f317CD861

**Comment by robertg2e3** [comment]
0xa97AEce59FaBbD22DC934ff61eA5Ba49f75e0F1A

**Comment by lixiaoping7769** [comment]
interesting.
0x0DAeB3424dcc83366B7d6BC08c7e05b58EE8ABD1

**Comment by leeshapping** [comment]
0xeb584436dBBb31D150BD96E1e2520b2E87440F6b
